src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -117,13 +117,22 @@ export const VetTecContactInformation = ({ institution }) => {
     primarySCOs.length > 0 && (
       <div className="vads-l-row vads-u-margin-y--4">
         <div className="vads-l-col--12 medium-screen:vads-l-col--3">
-          <h4 className="contact-heading">Primary</h4>
+          <h4
+            id="primary-contact-header"
+            className="contact-heading"
+            aria-label="primary contact officials"
+          >
+            Primary
+          </h4>
         </div>
         <div className="vads-l-col--9">
           <div className="vads-l-grid-container--full">
-            <div className="vads-l-row">
+            <ul
+              className="vads-l-row vettec-sco-list vads-u-margin--0"
+              aria-label="primary contact officials"
```

Comment:
This should be `aria-labelledby` and should reference the `<h4>` ID on line 121.